test-data/ert/poly_example/poly.ert

```diff
@@ -6,9 +6,11 @@ RUNPATH poly_out/realization-<IENS>/iter-<ITER>
 OBS_CONFIG observations
 REALIZATION_MEMORY 50mb
 
-NUM_REALIZATIONS 100
+NUM_REALIZATIONS 10
```

Comment:
Not sure if we should change anything in the test-data. Maybe we could instead just edit this in the test.

Reply:
this is just a boo-boo, will fix. Note to self, remove  `git commit -a` from muscle memory.

Reply:
No problem, it was just in the ready for review column so thought I would comment on it to be certain.

Reply:
fixed